src/openloco/game_commands.h

```diff
@@ -32,13 +32,15 @@ namespace openloco::game_commands
     }
 
     // Change company colour scheme
-    inline void do_19(int8_t ah, int8_t cl, int8_t dl)
+    inline void do_19(int8_t ah, int8_t al, uint8_t ax, int8_t cl, int8_t dh, int8_t dl)
     {
         registers regs;
         regs.ah = ah; // [ 0, 1 ] -- primary or secondary palette?
+        regs.al = al; // [ 0, 1 ] -- off or on
+        regs.ax = ax; // new colour?
```

Comment:
how can you have both al, ah and ax? Or is this different variables depending on dh?

Reply:
The latter, likely. It's currently trying to cover three code paths: 0x004332230 (dh = 1), 0x004331AF, 0x004331E9 (dh = 0)

Reply:
Okay. Either way i don't think this is doing what you think it's doing. You are basically ignoring al and ah as the ax overwrites al and ah.

Reply:
Oh, for sure, this is broken. I committed it in a WIP state yesterday as I was off to do other things.